src/shared/views/Login.js

```diff
@@ -140,6 +156,18 @@ type State = {
     signupPPChecked: boolean,
 };
 
+const checkInvalidUserName = name => {
+    if (name !== null && name.length < MIN_USERNAME_LENGTH) {
+        return true;
+    }
+
+    if (name !== null && name.length >= MIN_USERNAME_LENGTH) {
+        const validationRegex = /^[^A-Z\s]+$/;
+        return !validationRegex.test(name);
```

Comment:
Instead of regex validation let's transform the username and compare if it has changed.